src/DotVVM.Framework/Controls/GridViewValueColumn.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DotVVM.Framework.Binding;
+using DotVVM.Framework.Binding.Expressions;
+using DotVVM.Framework.Hosting;
+
+namespace DotVVM.Framework.Controls
+{
+    /// <summary>
+    /// A base class for GridView columns that present a value by wrapping a control.
+    /// </summary>
+    public abstract class GridViewValueColumn : GridViewColumn
+    {
+        /// <summary>
+        /// Gets or sets a binding which retrieves the value to display from the current data item.
+        /// </summary>
+        [MarkupOptions(AllowHardCodedValue = false, Required = true)]
+        public IValueBinding ValueBinding
+        {
+            get { return (IValueBinding)GetValue(ValueBindingProperty); }
+            set { SetValue(ValueBindingProperty, value); }
+        }
+        public static readonly DotvvmProperty ValueBindingProperty
+            = DotvvmProperty.Register<IValueBinding, GridViewValueColumn>(c => c.ValueBinding);
+
+        [MarkupOptions(AllowBinding = false)]
+        public bool IsControlValidated
+        {
+            get { return (bool)GetValue(IsControlValidatedProperty); }
+            set { SetValue(IsControlValidatedProperty, value); }
+        }
+        public static readonly DotvvmProperty IsControlValidatedProperty
+            = DotvvmProperty.Register<bool, GridViewValueColumn>(c => c.IsControlValidated, false);
+
+        [MarkupOptions(AllowBinding = false)]
+        public bool IsValidatorStandalone
```

Comment:
What if this is `true` while `IsControlValidated` is `false`? I guess there should be four valid states:
1. validation is diabled
2. The control has a validator attached.
3. There is the validator next to that control
4. Both of the above: the control has validator attached and message is displayed next to it.

This patch only seems to implement the first three, while the last one seems quite practical to me.